code/modules/shuttle/computer.dm

```diff
@@ -284,23 +284,36 @@
 
 /obj/structure/machinery/computer/shuttle/lifeboat/attack_hand(mob/user)
 	. = ..()
-	var/obj/docking_port/mobile/lifeboat/lifeboat = SSshuttle.getShuttle(shuttleId)
+	var/obj/docking_port/mobile/crashable/lifeboat/lifeboat = SSshuttle.getShuttle(shuttleId)
 	if(lifeboat.status == LIFEBOAT_LOCKED)
-		to_chat(user, SPAN_WARNING("\The [src] flickers with error messages."))
+		to_chat(user, SPAN_WARNING("[src] flickers with error messages."))
 	else if(lifeboat.status == LIFEBOAT_INACTIVE)
-		to_chat(user, SPAN_NOTICE("\The [src]'s screen says \"Awaiting evacuation order\"."))
+		to_chat(user, SPAN_NOTICE("[src]'s screen says \"Awaiting evacuation order\"."))
 	else if(lifeboat.status == LIFEBOAT_ACTIVE)
 		switch(lifeboat.mode)
 			if(SHUTTLE_IDLE)
-				to_chat(user, SPAN_NOTICE("\The [src]'s screen says \"Awaiting confirmation of the evacuation order\"."))
+				if(!istype(user, /mob/living/carbon/human))
+					to_chat(user, SPAN_NOTICE("[src]'s screen says \"Awaiting confirmation of the evacuation order\"."))
+					return
+
+				var/mob/living/carbon/human/human_user = user
+				if(!(ACCESS_MARINE_SENIOR in human_user.wear_id?.access))
```

Comment:
Perhaps a skillcheck for navigation would be more fitting? (Not that I have actually checked what difference it would make.)

Reply:
The only people with navigation skills are the CO, XO, ASO, CE, and Synths